releasenotes/notes/multi-layered-group-vars-cb60fcf454ca3bbe.yaml

```diff
@@ -0,0 +1,10 @@
+---
+other:
+  - |
+    RPC-Openstack had multiple levels of variable precedence, and we
+    heavily relied on user variables to complement/define
+    OpenStack-Ansible variables. This is not required anymore.
+    RPC-Openstack can now ship safe defaults for OpenStack-Ansible
+    without needing to set user variables. The user variables can
+    still be used by the deployer, but we now use the group_vars/
+    and host_vars/ folder for safe defaults in tree.
```

Comment:
I don't think that this note is explaining the value to the end-user (our support teams). How about something like this:

```
  - RPC-O previously relied on user-space configuration files to set variables used in playbooks across both the OpenStack-Ansible and RPC-O playbooks. These user-space configuration files provided global overrides only and could not be scoped to specific host groups. These defaults have now been set in group_vars in the code tree, reducing the need to overwrite user-space files, making them easier to scope to host groups, and reducing their precedence level to improve future development flexibility.
  - User-space configurable group_vars and host_vars can now optionally be used to customise deployments without requiring the user of extra-vars. The user-space group_vars and host_vars will take precedence over the in-tree group_vars. The user-space group_vars should be placed in /etc/openstack_deploy/host_vars and the user-space group_vars should be placed in /etc/openstack_deploy/group_vars.
```